.gitlab-ci.yml

```diff
@@ -83,6 +83,7 @@ update-cache: &update-cache-manual
     - me_dev
     - top_dev
     - sw_dev
+    - me_dev_fix_interfacerename
```

Comment:
I assume we will pull this out from me_dev post merge?

Reply:
yep!